src/utils/services/calendar.helper.ts

```diff
@@ -58,8 +59,13 @@ export const CalendarServiceHelper = {
 
     message += meetingLocationText
 
-    if (meetingChangeLink) {
-      message += `\n\nTo reschedule or cancel the meeting, please go to ${meetingChangeLink}`
+    if (rescheduleUrl || cancelUrl) {
+      if (rescheduleUrl) {
+        message += `\n\nTo reschedule the meeting, please go to: ${rescheduleUrl}`
```

Comment:
Instead, we could say to reschedule or cancel the meeting, go to: reschedule link because a user can also cancel the meeting there so there's no need to make the copy any longer